packages/three-vrm-springbone/src/VRMSpringBoneJoint.ts

```diff
@@ -83,6 +83,26 @@ export class VRMSpringBoneJoint {
    */
   private _worldSpaceBoneLength = 0.0;
 
+  /**
+   * List of dependencies that need to be updated before this joint.
+   */
+  public get dependencies(): THREE.Object3D[] {
```

Comment:
[SHOULD] Existing VRMConstraints already have signatures `get dependencies(): Set<THREE.Object3D>`. I want to align the interface unless it needs to be an array instead.

Reply:
Initially I moved it as I wanted to cache the list of dependencies, since it caused a lot of memory allocations in the hot-path (`VMRSpringBoneManager.update`). The `VRMSpringBoneJoint` itself seemed like a good place to keep the cached list as it could also invalidate it when needed. But after pre-computing the joint order, it was no longer in the hot-path, so I removed the caching again.

In the end the move wasn't needed. But I like the consistency with the `VRMConstraints`, so I'll update the signature to return a `Set<THREE.Object3D>` as well.